extensions/legit/legit-commit.lisp

```diff
@@ -92,9 +92,9 @@ two
       ((str:blankp cleaned-message)
        (message "No commit message, do nothing."))
       (t
-       (with-current-project ()
+       (with-current-project vcs
```

Comment:
I would prefer we use a list of parameters as in:

```lisp
(with-current-project (vcs)
  …)
```

it's the form seen most often IME, it's also the one most future-proof, we can add params alongside the vcs.